recipe/meta.yaml

```diff
@@ -92,6 +92,55 @@ outputs:
         Compiler for CUDA applications.
       doc_url: https://docs.nvidia.com/cuda/index.html
 
+  - name: cuda-crt
```

Comment:
Should we also add a `cuda-crt-dev` either here or in `cuda-nvcc-impl`, which pulls in `cuda-crt-dev_{{ target_platform }}`? This may also be a bit more ergonomic

This would be somewhat similar to what we did with [`cuda-cudart-dev`]( https://github.com/conda-forge/cuda-cudart-feedstock/blob/a317588c669cde9dee58ccc8bb0cd22c69d4a4a6/recipe/meta.yaml#L112 ). Though that has a few more pieces to pull in

Understand if this feels redundant. Mainly wanted to surface the suggestion so we could make a decision either way

Reply:
I don't have strong opinions here. I am okay with the current implementation.

Reply:
Same, no strong opinion here. All of these details should be hidden from the user anyway, so I'd say we hold off on adding more packaging indirection until we see more benefits for us as maintainers. Right now I think the tradeoff of clarity for volume is pretty even.